introduction.md

```diff
@@ -0,0 +1 @@
+Hi. Well I don't want to say too much. I am that shy.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.